drivers/regulator/regulator_da1469x.c

```diff
@@ -369,6 +390,37 @@ static int regulator_da1469x_init(const struct device *dev)
 	return regulator_common_init(dev, 0);
 }
 
+#if IS_ENABLED(CONFIG_PM_DEVICE)
+static int regulator_da1469x_pm_action(const struct device *dev,
+				       enum pm_device_action action)
+{
+	const struct regulator_da1469x_config *config = dev->config;
+	int ret = 0;
+
+	switch (action) {
+	case PM_DEVICE_ACTION_RESUME:
+		if (config->desc->dcdc_register) {
+			*config->desc->dcdc_register = *config->desc->dcdc_register_shadow;
+			if ((CRG_TOP->ANA_STATUS_REG & CRG_TOP_ANA_STATUS_REG_COMP_VBAT_HIGH_Msk) &&
+			    (*config->desc->dcdc_register_shadow & DCDC_REQUESTED)) {
+				DCDC->DCDC_CTRL1_REG = dcdc_state.ctrl1;
+			}
+		}
+		break;
+	case PM_DEVICE_ACTION_SUSPEND:
+		/*
+		 * Shadow register is saved on each regulator API call, there is no need
+		 * to save it here.
+		 */
+		break;
+	default:
+		ret = -ENOTSUP;
+	}
+
+	return ret;
+}
+#endif
```

Comment:
do you really need this if notifier bit is in place?

Reply:
removed

Reply:
where is this removed?